content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md

```diff
@@ -0,0 +1,347 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+title: Upgrading kubeadm clusters from v1.13 to v1.14
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page explains how to upgrade a Kubernetes cluster created with `kubeadm` from version 1.13.x to version 1.14.x, and from version 1.14.x to 1.14.y, where `y > x`.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+- You need to have a `kubeadm` Kubernetes cluster running version 1.13.0 or later.
+  [Swap must be disabled][swap].
+  The cluster should use a static control plane and etcd pods.
+- Make sure you read the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md) carefully.
+- Make sure to back up any important components, such as app-level state stored in a database.
+  `kubeadm upgrade` does not touch your workloads, only components internal to Kubernetes, but backups are always a best practice.
+
+
+[swap]: https://serverfault.com/questions/684771/best-way-to-disable-swap-in-linux
+### Additional information
+
+- All containers are restarted after upgrade, because the container spec hash value is changed.
+- You can upgrade only from one minor version to the next minor version.
+  That is, you cannot skip versions when you upgrade.
+  For example, you can upgrade only from 1.10 to 1.11, not from 1.9 to 1.11.
+
+{{< warning >}}
+The command `join --experimental-control-plane` is known to fail on single node clusters created with kubeadm v1.12 and then upgraded to v1.13.x.
+This will be fixed when graduating the `join --control-plane` workflow from alpha to beta.
+A possible workaround is described [here](https://github.com/kubernetes/kubeadm/issues/1269#issuecomment-441116249).
+{{</ warning >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Determine which version to upgrade to
+
+1.  Find the latest stable 1.14 version:
+
+    {{< tabs name="k8s_install_versions" >}}
+    {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+    apt update
+    apt-cache policy kubeadm
+    # find the latest 1.14 version in the list
+    # it should look like 1.14.x-00, where x is the latest patch
+    {{% /tab %}}
+    {{% tab name="CentOS, RHEL or Fedora" %}}
+    yum list --showduplicates kubeadm --disableexcludes=kubernetes
+    # find the latest 1.14 version in the list
+    # it should look like 1.14.x-0, where x is the latest patch
+    {{% /tab %}}
+    {{< /tabs >}}
+
+## Upgrade the control plane node
+
+1.  On your control plane node, upgrade kubeadm:
+
+    {{< tabs name="k8s_install_kubeadm" >}}
+    {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+    # replace x in 1.14.x-00 with the latest patch version
```

Comment:
that -00 versioning doesn't look right to me.

Reply:
this is something we decided on last cycle
cc @mauilion